test/OP2BitmapConverterTest.vcxproj

```diff
@@ -44,6 +44,9 @@
     <ProjectReference Include="..\OP2BitmapConverter.vcxproj">
       <Project>{c3e96353-8151-405c-9611-0bb0fd76ea74}</Project>
     </ProjectReference>
+    <ProjectReference Include="..\OP2Utility\OP2Utility.vcxproj">
+      <Project>{980d53d9-f9e2-4682-9307-1303c9e42313}</Project>
+    </ProjectReference>
```

Comment:
I'm using OP2Utility to load the bitmaps into memory and compare if them. The following lines of code require a reference to OP2Utility:

```C++
auto bitmap1 = Tileset::ReadTileset(Stream::FileReader(startingTilesetFilename));
auto bitmap2 = Tileset::ReadTileset(Stream::FileReader(standardTilesetFilename));

ASSERT_EQ(bitmap1, bitmap2);
```

Now that I am thinking about it based on your comment, we don't need to specifically load the files as bitmaps to compare them. Maybe we can find a platform agnostic solution to compare equality of contents of two files on disk, either directly as files or load them into generic memory blocks and compare?

That would allow pulling the reference to the OP2Utility.

Reply:
Ahh, I see. That makes sense then.

My second thought there was also to not parse the data as bitmaps to compare them. Though reading from disk might still want to make use of the `Stream` API. I suppose you could do it with raw C++ calls. Don't know if it matters really.

Anyway, I'm fine with the reference. I think I just wanted to be clear on why it was being added.

Reply:
@DanRStevens,

I poked around the internet a bit and didn't any sort of quick one line solution for reading the data and comparing using the std library. 

I took a shot at using our Stream code and the following passes the unit test. I kind of like the shorter syntax of loading it like a bitmap that is already committed. If you think below is valuable, I can commit it though.

```C++
	Stream::FileReader reader1(startingTilesetFilename);
	std::vector<uint8_t> bitmap1;
	bitmap1.resize(reader1.Length());
	reader1.Read(bitmap1.data(), reader1.Length());

	Stream::FileReader reader2(standardTilesetFilename);
	std::vector<uint8_t> bitmap2;
	bitmap2.resize(reader2.Length());
	reader2.Read(bitmap2.data(), reader2.Length());

	ASSERT_EQ(bitmap1, bitmap2);
```

Reply:
I think I would prefer the more compact solution you already have.

This does get me wondering though, if there a way we could improve the stream API to make it easy to read the contents into a `std::string` or `std::vector`. Or perhaps compare two streams directly. That would be more of an OP2Utility task though.

----

In terms of standard library solutions, I just took a quick look at:
https://stackoverflow.com/questions/2602013/read-whole-ascii-file-into-c-stdstring

It's not super pretty, though I suppose it's not terribly different from the `Stream` approach in your comment.

----

I suppose a utility method could be written that encapsulates the idea of code you just wrote in your comment:
```cpp
std::string ReadFile(fileName);
```

```cpp
ASSERT_EQ(ReadFile(startingTilesetFilename), ReadFile(standardTilesetFilename));
```